hs_core/views/__init__.py

```diff
@@ -1079,7 +1079,6 @@ def create_resource_select_resource_type(request, *args, **kwargs):
 @login_required
 def create_resource(request, *args, **kwargs):
     # Note: This view function must be called by ajax
-
     ajax_response_data = {'status': 'error', 'message': ''}
```

Comment:
I know it seems a little silly...but removing these errant spaces from __init__.py completely would be helpful merge wise later on.